parsec/core/mountpoint/manager.py

```diff
@@ -268,15 +268,18 @@ async def safe_unmount_all(self):
 
 
 async def cleanup_macos_mountpoint_folder(base_mountpoint_path):
-    # In case of a crash on macOS, workspaces don't unmount correctly and leave empty directories
-    # in the default mount folder. This function is used to clean these anytime a login occurs.
+    # In case of a crash on macOS, workspaces don't unmount correctly and leave
+    # empty mountpoints or directories in the default mount folder. This
+    # function is used to unmount or delete these anytime a login occurs.
     for dirs in os.listdir(base_mountpoint_path):
```

Comment:
`dirs` ending with a `s` suggest it is an array, what about `mountpoint_name` instead (and `mountpoint_path` instead of `dir_path`) ?